Start time: 06:55
dataset: spacev10m
N: 10000000
dataset file: /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs
query file: /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs
attr file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json
query range file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json
ground truth file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json
K: 10
nprobe: 50
collection name: collection_spacev10m_sel_1_500_random_100
2025-04-09 21:56:01,989 [ERROR][_create_connection]: Failed to create new connection using: 186f214a5cbc442a9bfc0d696da8c170 (milvus_client.py:923)
Traceback (most recent call last):
  File "utils/pymilvus_ivfpq.py", line 67, in <module>
    client = MilvusClient(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 67, in __init__
    self._using = self._create_connection(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 924, in _create_connection
    raise ex from ex
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 921, in _create_connection
    connections.connect(using, user, password, db_name, token, uri=uri, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 461, in connect
    connect_milvus(**kwargs, user=user, password=password, token=token, db_name=db_name)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 411, in connect_milvus
    gh._wait_for_channel_ready(timeout=timeout)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 155, in _wait_for_channel_ready
    raise MilvusException(
pymilvus.exceptions.MilvusException: <MilvusException: (code=2, message=Fail connecting to server on localhost:19530, illegal connection params or server unavailable)>
Command exited with non-zero status 1
	Command being timed: "python -u utils/pymilvus_ivfpq.py spacev10m 10000000 /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json 10 50 collection_spacev10m_sel_1_500_random_100 query 100 100"
	User time (seconds): 2.12
	System time (seconds): 0.13
	Percent of CPU this job got: 21%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:10.74
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 75776
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 24
	Minor (reclaiming a frame) page faults: 22188
	Voluntary context switches: 2048
	Involuntary context switches: 445
	Swaps: 0
	File system inputs: 2440
	File system outputs: 0
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
Start time: 06:56
dataset: spacev10m
N: 10000000
dataset file: /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs
query file: /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs
attr file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json
query range file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json
ground truth file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json
K: 10
nprobe: 50
collection name: collection_spacev10m_sel_1_500_random_100
collection  collection_spacev10m_sel_1_500_random_100  exists
collection size: data: ["{'count(*)': 10000000}"]
query d: 100
get query szie: 10000
get range cnt: 10000
ngt= 10000  nq= 10000  k= 10
ids shape: 10000 10
gt shape: (10000, 10)
milvus get  72021  postive res from  100000  results, recall@ 10 : 0.72021
recall dist(10\% per bucket):  [0.0034 0.0093 0.0183 0.0405 0.0573 0.0812 0.1204 0.1528 0.1799 0.2007
 0.1362]
10000  queries cost: 89.4159038066864 , qps: 111.83692804380303
	Command being timed: "python -u utils/pymilvus_ivfpq.py spacev10m 10000000 /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json 10 50 collection_spacev10m_sel_1_500_random_100 query 100 100"
	User time (seconds): 21.70
	System time (seconds): 2.68
	Percent of CPU this job got: 20%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 1:56.96
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 92924
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 26
	Minor (reclaiming a frame) page faults: 60986
	Voluntary context switches: 24622
	Involuntary context switches: 528734
	Swaps: 0
	File system inputs: 4536
	File system outputs: 24
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
